tests/test_check.py

```diff
@@ -61,6 +61,35 @@ def test_check_on_file():
         )
 
 
+def test_check_on_package():
+    """
+    Test verifies if `check_all()` works with package notation (dots) and also mixed with raw file paths.
+
+    Input mixes both package and package files covering all cases
```

Comment:
This line is not necessary. The first part repeats information from the first line; the second, "covering all cases", does not convey useful information. In general comments and docstrings shouldn't comment on the quality of the code, as this should be illustrated by the code itself.